aiida/cmdline/params/types/profile.py

```diff
@@ -9,10 +9,10 @@
 ###########################################################################
 """Profile param type for click."""
 
-import click
+from .strings import LabelStringType
 
 
-class ProfileParamType(click.ParamType):
+class ProfileParamType(LabelStringType):
```

Comment:
Here we are making the rules more strict. There might be profiles out there that don't respect the rules of `LabelStringType`. Those will no longer be able to be used in the CLI. Of course, the profile name can easily be adapted manually in the `config.json`, just wanted to check if you thought about this. If you are ok with doing this, then also fine by me

Reply:
I'm aware, but I think this change is still reasonable. My arguments:
 * as you say, it's something that can be easily fixed (and it's probably in the best interest of people to do so)
 * it can prevent issues during profile setup, which is the first thing new aiida users do
 * the profile name is not exported, so this does not invalidate any published data